README.md

```diff
@@ -87,6 +87,17 @@ INSERT INTO "schema_migrations" (version) VALUES
 ;
 ```
 
+By default the gem will remove some extensions that typically aren't needed in structure dumps. You can choose to keep all, or just some, of those extensions:
+
+```ruby
+Rails.application.configure do
+  config.activerecord_clean_db_structure.keep_extensions = :all
+
+  # This does the same thing as :all. You can choose which optional extensions to keep.
+  config.activerecord_clean_db_structure.keep_extensions = ["pg_stat_statements", "pg_buffercache"]
```

Comment:
Should we say "put these changes in `config/application.rb`" or is that obvious enough?

Reply:
It can also go in an initializer.  I think anyone using this gem probably understands how to use rails config.